receiver/signalfxreceiver/go.mod

```diff
@@ -80,9 +80,6 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
-// TODO: remove once the next release of jaeger is out
-replace github.com/jaegertracing/jaeger v1.39.0 => github.com/jaegertracing/jaeger v1.39.1-0.20221110195127-14c11365a856
```

Comment:
Should this PR also upgrade the dependency?

Reply:
The next dependabot PR will update the jaeger dependency anyways, if you can approve i'll create that PR next.

Reply:
Ok, I just wanted to make sure it's not left with this version